SiteOutline.md

```diff
@@ -0,0 +1,119 @@
+# DLF AIG Metadata Working Group Site Outline
+
+This is the proposed reorganization of the DLF AIG Metadata Working Group output. This outline is to ensure we cover all work that is currently being done, as well as to help us plan our Jekyll changes/migration.
+
+The .md / .html extensions to propose pages have been removed, as they will be dependent on the Jekyll migration regardless. Instead, the bolded portions are just indicated to be pages, further form (stand-alone html, post, include, other) to be determined.
+
+## **index**: Introduction, Scoping, Overview (2016-2017)
+
+Information on **index**, which is ultimately the homepage:
```

Comment:
This could get very long. What do we think about moving some of this to its own page (an About page)?